permissions/plugin-defensics.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: "defensics"
+github: "jenkinsci/defensics-plugin"
+paths:
+- "com/synopsys/defensics"
+developers:
+- "synhal"
+- "defensics_oss"
+security:
+  contacts:
+    jira: defensics_oss
```

Comment:
```suggestion
```

Since this is the same name as one of the developers, it is not needed.

Reply:
Thanks, I removed it. 

The sentence "Given the above example, we will assign any security issue in Jira to some_user_name rather than one of the developers able to release the plugin" in this repo's README lead me to think you might contact a random developer if left out, and we wanted to make sure the correct account would be contacted. I put defensics_oss first, just in case.